packages/web/src/components/client-only/ClientOnly.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactNode, useEffect, useState } from 'react'
+
+type ClientOnlyProps = {
+  children: ReactNode
+  fallback?: ReactNode
+}
+
+/**
+ * A simple wrapper that only renders its children on the client.
+ * This is used to disable certain parts of the app from rendering on the server.
+ *
+ * If you need to lazy load something on the client only, use vike-react/ClientOnly
+ */
+export const ClientOnly = (props: ClientOnlyProps) => {
+  const { children, fallback = null } = props
+
+  const [isClientSide, setIsClientSide] = useState(false)
```

Comment:
i guess this answers my q previously, so could we just use this in that one page where we manually check "isClientSide"?

Reply:
That one is in stems, didn't want to add a `ClientOnly` component there just for a single case